docker-compose-rule-core/src/main/java/com/palantir/docker/compose/connection/waiting/Exceptions.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.docker.compose.connection.waiting;
+
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+
+public enum Exceptions {
+    ;
```

Comment:
:(

Reply:
Hey, the "organically reared zero variant enum with side of flambé hanging semicolon" is a classic ~~dish~~coding style!